testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -1238,10 +1238,10 @@
 end
 
 When(/^I enter "([^"]*)" hostname on grafana's host field$/) do |host|
-  step 'I click on "var-hostname"'
+  grafana_url = current_url
   system_name = get_system_name(host)
-  step %(I enter "#{system_name}" as "Enter variable value")
-  send_keys(:return)
+  updated_url = grafana_url.sub(/var-hostname=[^&]*/, "var-hostname=#{system_name}")
+  visit updated_url
```

Comment:
I preferred previous implementation because it was not using knowledge about internals, but rather behaving as an user interacting with the UI.

Reply:
Maybe we should move from Selenium to API, using their REST API
https://grafana.com/docs/grafana/latest/developers/http_api/